File: python/telco_system/telco_system.py
class TelcoSystem:
    def __init__(self):
        raise NotImplementedError()

    @staticmethod
    def with_peak_hour_starting_at(an_hour):
        raise NotImplementedError()

    def register_international_call_between(self, start_date, end_date, client_name):
        raise NotImplementedError()

    def register_national_call_between(self, start_date, end_date, client_name):
        raise NotImplementedError

    def register_local_call_between(self, start_date, end_date, client_name):
        raise NotImplementedError()

    def historical_total_billed(self):
        raise NotImplementedError()

    def historical_number_of_calls(self):
        raise NotImplementedError()

    def total_billed_for(self, client_name):
        raise NotImplementedError()

    def number_of_calls_for(self, client_name):
        raise NotImplementedError()

File: php/telco_system/src/TelcoSystem.php
<?php


class TelcoSystem
{
    public static function withRushHourStartingAt(int $int)
    {
        throw new Exception("Not implemented");
    }

    function historicalTotalBilled()
    {
        throw new Exception("Not implemented");
    }

    public function historicalNumberOfCalls()
    {
        throw new Exception("Not implemented");
    }

    public function registerInternationalCallBetween($startTime, $endTime, $clientName)
    {
        throw new Exception("Not implemented");
    }

    public function registerNationalCallBetween($startTime, $endTime, $clientName)
    {
        throw new Exception("Not implemented");
    }

    public function registerLocalCallBetween($startTime, $endTime, $clientName)
    {
        throw new Exception("Not implemented");
    }

    public function numberOfCallsFor(string $clientName)
    {
        throw new Exception("Not implemented");
    }

    public function totalBilledFor(string $clientName)
    {
        throw new Exception("Not implemented");
    }
}
